infer/lib/audio.py

```diff
@@ -16,9 +16,9 @@
 
 def wav2(i: BytesIO, o: BufferedWriter, format: str):
     inp = av.open(i, "r")
-    format = video_format_dict[format]
+    format = video_format_dict.get(format) or "mp4"

```

Comment:
Well, just `get(format, format)`, if my memory is correct.

Reply:
Huh, I didn't know Python supported that
Will change.